airbyte-cdk/java/airbyte-cdk/db-destinations/src/testFixtures/kotlin/io/airbyte/cdk/integrations/standardtest/destination/TestingNamespaces.kt

```diff
@@ -38,7 +38,7 @@ object TestingNamespaces {
      *
      * @return convention-compliant namespace
      */
-    @JvmOverloads
+    @JvmStatic
     fun generate(prefix: String? = null): String {
```

Comment:
Generated decided to use `@JvmOverloads` for static overloaded methods. Maybe a Companion object with default params and `@JvmOverloads` might be better here ?

Reply:
I think that the Kotlin convention would be to use the companion object, but I don't believe that there is really any technical difference.  I think its more about readability and is really only temporary as long as we have Java code that needs to call this/will go away when that code becomes Kotlin too.

Reply:
the @JvmOverload says that the compiler will generate 2 static `generate` functions: one with no parameter, and one with 1 parameter. Has little to do with static.
You really want `@JvmOverload @JvmStatic`

Reply:
For this method to be used in java's static context like `TestingNamespaces.generate()` && `TestingNamespace.generate("string")` had to use `@JvmStatic`. `@JvmOverloads` works with instance methods. Its a different story if used in Kotlin since kotlin supports `TestingNamespaces.generate()` call on singleton `object`

Reply:
I actually tested this. If the function `generate(prefix: String? = null): String` is annotated with `@JvmStatic` and `@JvmOverloads`, and the function `generate()` is removed, then the compiled will create a JVM `static public generate()` function. Let's do that

Reply:
Interesting. double annotate you mean. Will try that.